### PR TITLE
[boost-multi-array] Additional dependencies for boost multiarray

### PR DESCRIPTION
**Pull request by vmiheer**
**Describe the pull request**

- What does your PR fix? Fixes
dependency for boost multiarray

- Which triplets are supported/not supported? Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

File: ports/boost-multi-array/CONTROL [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Automatically generated by boost-vcpkg-helpers/generate-ports.ps1
 Source: boost-multi-array
 Version: 1.73.0
```
Comment (PhoebeHui) [inline ports/boost-multi-array/CONTROL:3]:
```suggestion
Version: 1.73.0
Port-Version: 1
```

**Review by PhoebeHui — COMMENTED** [review]
@vmiheer, thanks for the PR!

File: ports/boost-multi-array/CONTROL (hunk 1 above)
Comment (vmiheer) [reply ports/boost-multi-array/CONTROL:3]: Thank you!

**Review by PhoebeHui — APPROVED** [review]

**Comment by dan-shaw** [comment]
What issue does this fix?

**Comment by vmiheer** [comment]
If I were to use boost multiarray, IMO, `vcpkg install boost-multi-array` followed by use of it in a project using cmake should work out of the box. But without this patch, my compilation fails. It includes other headers e.g. boost-array https://github.com/boostorg/multi_array/blob/3b405720559ff9acc968e1a40102ca6245a64e6c/include/boost/multi_array.hpp#L38. So I think when I do `vcpkg install boost-multi-array` it should get all other necessary header/packages too.

**Comment by vmiheer** [comment]
@dan-shaw ping!

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**